dataland-frontend/src/components/forms/parts/BasicInformationFields.vue

```diff
@@ -0,0 +1,136 @@
+<template>
```

Comment:
Sollte die Komponente irgendwie euTaxonomy im Namen haben?

Reply:
I'm starting to work on it.

Reply:
component renamed to EuTaxonomyBasicInformation

Reply:
ok